src/Jackett/Jackett.csproj

```diff
@@ -444,6 +445,7 @@
     <Content Include="Content\login.html">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="Content\logos\myanonamouse.png" />

```

Comment:
Error in your code:
Must include _Copy to Output Directory instruction_

See after for usage ;)